packages/roosterjs-editor-types/lib/interface/SelectionRangeEx.ts

```diff
@@ -0,0 +1,53 @@
+import IVTable from './IVTable';
+
+/**
+ * Represents the selection made inside of a table.
+ */
+export interface ITableSelectionRange
+    extends ISelectionRangeEx<SelectionRangeTypes.TableSelection> {
+    vTable: IVTable;
+}
+
+/**
+ * Represents normal selection
+ */
+export interface INormalSelectionRange extends ISelectionRangeEx<SelectionRangeTypes.Normal> {}
+
+/**
+ * Represents normal selection
+ */
+export interface ISelectionRangeEx<T> {
```

Comment:
Let's remove the prefix "I", as long as it is not conflict with existing types. Same for other interfaces

Reply:
This one have conflict with the union type setting it as SelectionRangeExBase